documentation/design-docs/dotnet-tools.md

```diff
@@ -282,19 +282,23 @@ COLLECT
         asp-net-basic   Useful starting point for ASP.Net performance investigations
         gc              Tracks allocation and collection performance
         gc-collect      Tracks GC collection only at very low overhead
-        none            Tracks nothing. Only providers specified by the --provider option will be available.
```

Comment:
Is the removal of the none option intentional? If someone creates a custom EventSource and they want to trace that output of that one source and nothing else, do they have a way to achieve that?

Reply:
Nm, I messed up reading the diff : )